src/splitInteger.js

```diff
@@ -4,6 +4,7 @@
  *
  * @returns {number[]}
  */
+// eslint-disable-next-line strict
 function splitInteger(value, numberOfParts) {
```

Comment:
The 'strict' mode is disabled which can result in unexpected behaviors. It is recommended to always use 'strict' mode in JavaScript.